config/locales/exceptions.en.yml

```diff
@@ -130,6 +130,81 @@ en:
     # Below this line just define the minor code as key to be used by client error
       VA900:
         <<: *external_defaults
+      SM105: # What casues this? Perhaps then we can come up with more customized detail below
+        <<: *external_defaults
+        code: 'SM105'
+        detail: 'User was not found'
+        status: 422
+      SM112: # 'User is not owner of entity requested' (usually attachments)
+        <<: *external_defaults
+        code: 'SM112'
+        detail: 'You do not have access to the requested resource'
+        status: 403
+      SM115:
+        <<: *external_defaults
+        code: 'SM115'
+        title: Record not found
+        detail: 'Attachment requested could not be found'
+        status: 404
+      SM116: # 'The Folder must be empty before delete'
+        <<: *external_defaults
+        code: 'SM116'
+        detail: 'Folder must be empty before you can delete it'
+        status: 422
+      SM119: # 'Triage team does not exist' when trying to create new message
+        <<: *external_defaults
+        code: 'SM119'
+        detail: 'Triage team does not exist'
+        status: 422
+      SM129:
+        <<: *external_defaults
+        code: 'SM129'
+        detail: 'Unable to reply because you are no longer associated with this Triage Team'
+        status: 422
+      SM130:
+        <<: *external_defaults
+        code: 'SM130'
+        detail: 'Unable to reply because the source message is expired'
+        status: 422
+      SM135: # 'User is not eligible because they have not accepted terms and conditions or opted-in'
+        <<: *external_defaults
+        code: 'SM135'
+        detail: 'You have not accepted the MHV T&C to use secure messaging'
```

Comment:
I would avoid having abbreviations/jargon in the exception message - i.e. expand to "terms and conditions"

If we ever do actually translate these, translation team will need this expanded anyway.